Gemfile

```diff
@@ -1,11 +1,16 @@
 source "https://rubygems.org"
 gemspec
 
+gem "aws-sdk-kafka", "~> 1.68"
+gem "base64", "~> 0.2"
 gem "minitest", "~> 5.11"
 gem "minitest-rg", "~> 5.3"
+gem "minitest-stub_any_instance", "~> 1.0"
 gem "rake", "~> 13.0"
 gem "rubocop", "1.60.2"
 gem "rubocop-minitest", "0.34.5"
 gem "rubocop-packaging", "0.5.2"
 gem "rubocop-performance", "1.20.2"
 gem "rubocop-rake", "0.6.0"
+gem "simplecov", require: false, group: :test
+gem "thor", "~> 1.3"
```

Comment:
Don't put this here since it will be read from gemspec above

Reply:
fixed